app/jobs/cluster_status_job.rb

```diff
@@ -5,12 +5,11 @@ class << self
   end
 
   def perform(cluster_id)
-    cluster = ActiveJobs.clusters[cluster_id.to_sym]
-    info = ActiveJobs.info_from_jobs(cluster.job_adapter.info_all)
```

Comment:
I started to inspect what was up with caching and found these are on different code paths. So a little bit of this change is to ensure that `all` is cached correctly and that all APIs/consumers use `all` for it's cached values.